presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -89,6 +89,10 @@ Property                                                Description
                                                         used to secure TLS.
 ``http-server.https.keystore.key``                      The password for the keystore. This must match the
                                                         password you specified when creating the keystore.
+``http-server.authentication.forwardHttps``             Enable treating forwarded HTTPS requests over HTTP
```

Comment:
How about `http-server.authentication.allow-forwarded-https`?

Reply:
done, I agree that it reads better